inception-doc/src/main/resources/META-INF/asciidoc/user-guide/getting-started.adoc

```diff
@@ -1,79 +1,399 @@
+defined set of values to enter. You can modify and create tagsets in the
+project settings (section „Tagsets“) which we will deal with later. See
+our main documentation on how to create and modify tagsets. LINK
+EINFÜGEN
+* In the **Annotation Sidebar **at the left (it folds out when clicking
+on the little arrow on top) there are several features you might want to
+check the main documentation for. Especially the *Recommender* section
+(the black speech bubble) is worth a look if you use recommenders
+(recommenders will be explained later on in this guide). E.g. you find
+their measures and learning behaviours. Note the *Search* (the magnifier
+glass): You can create or delete annotations on all or some of the
+search results.
+* To get familiar with INCEpTION, you may want to follow the
+instructions for your example-project which are on the website where you
+downloaded it or explore INCEpTION yourself, learning by doing.
++
+One way or the other: *Have fun exploring*!
 
-== Where to go from here
+Anatomy of an annotation project
 
-To familiarize yourself with the functionalities of {product-name}, try importing some of the link:https://inception-project.github.io//use-case-gallery/[{product-name} example projects]. 
+* In order to work with your own projects, you might want to understand
+the anatomy of an INCEpTION project. Let‘s see what the dashboard
+(clicking on the dashboard button on the top) provides: As a Project
+Manager of a project, you see all of the following sub pages (roles and
+rights will be presented later on in this guide). For details on each
+section, check the main documentation.
+** *Annotation*: We have been here already. Here, you can make your
+annotationsjust loke before.
+** *Curation*: If several annotators work on a project, you may want a
+curator to finally decide which annotations to accept. Everyone with
+curation rights (see below) within this project can curate. All others
+do not have access to nor see this page. Only documents marked as
+finished by at least one annotator can be curated. For details on how to
+courate, see the main documentation LINK EINFÜGEN or just try it out:
+*** Add another fake user (see below)
+*** Create some annotations in any document
+*** Mark the document as finished: Just click on the lock on top.
+*** Log out and log in again as the fake user.
+*** In the very same document, make some annotations exactly like before
+but also some which are different. Mark the document as finished.
+*** Log in as any user with couration rights (e.g. the „admin“ user we
+used before), enter the couration page and explore how to courate: You
+see the automatic merge on top (what was equal from both users is being
+accepted already) and the annotations of each of the users below.
+Differences are highlightet. You can accept an annotation by clicking on
+it. You can also create new annotations.
+** *Knowledge Base*: On this sub page, you can manage and create your
```

Comment:
I would explain in one sentence what KBs are

Reply:
box inserted